src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -43,6 +50,15 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       var chatDiv = document.getElementById('chat');
       chatDiv.scrollTop = chatDiv.scrollHeight;
     };
+
+    function toggleReply(name) {
+    var x = document.getElementById("replyForm");
+    if (x.style.display == "none") {
```

Comment:
It looks like this isn't being used yet. Remove for now and include it in a followup pull-request? When you do have this working, consider using something like x.classList.replace("hide", "show") to control visibility via CSS classes instead of directly modifying the style.